pkg/virt-launcher/virtwrap/converter/archconverter.go

```diff
@@ -40,10 +41,21 @@ func newArchConverter(arch string) archConverter {
 		return archConverterARM64{}
```

Comment:
I agree that having multiarch code restricted to arch specific files would look even better, but i thing given that we need the interface definition anyway, having a central file dedicated to being the gateway to multiple architectures is a good compromise that allows us to not need to deal with nil pointer exceptions when calling arch specific code.

Besides, the good thing about switch-case is that it is not a really a chain like if-else.

For the other instances of arch specific code, at least from what i know was mostly single string values and other small things. With the discussed arch owner groups (https://github.com/kubevirt/community/pull/298) it might make more sense to put these into separate files. If that is wished, i can raise that as a follow up PR, but i would not put it in here, as that is a separate change and would need https://github.com/kubevirt/community/pull/298 to be implemented first anyway.